packages/core/src/store/actions.ts

```diff
@@ -71,13 +74,35 @@ export function addChains(chains: Chain[]): void {
       ...rest,
       namespace,
       id: id.toLowerCase(),
-      rpcUrl: rpcUrl.trim()
+      rpcUrl: rpcUrl ? rpcUrl.trim() : null
     }))
   }
 
   dispatch(action as AddChainsAction)
 }
 
+export function updateChain(updatedChain: Chain): void {
```

Comment:
Is documentation for this function added to the core README and docs markdown?
There is a section around api functions

Reply:
We will also want to add a wrapper around this to the React package, maybe the Vue as well and then add documentation for it to eh React README along with the React docs md

Reply:
@Adamj1232 does this function need to be separately documented in the core README? I've updated the documentation around the `setChain` function to include it will not accept an rpcUrl, token, and label. 
`updateChain` is currently used as part of the `setChain` function. 
`updateChain` isn't currently an exported function. Should I change this to make it available on it's own as part of the package, instead of just called inside the `setChain` function?

Reply:
Ohhh ahh you are correct then! All good!! No need to expose unless requested or needed. More coffee required on my end lol